benches/coprocessor_executors/mod.rs

```diff
@@ -11,14 +11,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(clippy::blacklisted_name)]
+
 #[macro_use]
```

Comment:
Could we import macro via use statements?

Reply:
We probably could. THe changes are the ones I could make semi-mechanically with Rustfix, there are probably better ways